### PR TITLE
Add list all users data

### PR DESCRIPTION
**Pull request by omgronny**

### CODE REVIEW

File: src/main/java/org/itmo/eventapp/main/model/mapper/UserMapper.java [hunk 1]
```diff
@@ -19,4 +23,21 @@ public static UserShortDataResponse userToUserShortResponse(User user) {
                 user.getSurname()
         );
     }
+
+    public static UserSystemRoleResponse userToUserSystemRoleResponse(User user) {
+        return new UserSystemRoleResponse(
+                // getUserLoginInfo is always not null
```
Comment (omgronny) [inline src/main/java/org/itmo/eventapp/main/model/mapper/UserMapper.java:29]: кстати, это же правда?

**Comment by IP-13** [comment]
|Overall Project|70.13%|:green_apple:|
|:-|:-|:-:|
|Files changed|100%|:green_apple:|
<br>

|File|Coverage||
|:-|:-|:-:|
|[ProfileController.java](https://github.com/itmo-events-app/event-app/blob/ba72ae724aa667cb1e3684710be0c27903922553/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fcontroller%2FProfileController.java)|100%|:green_apple:|
|[UserSecurityExpression.java](https://github.com/itmo-events-app/event-app/blob/ba72ae724aa667cb1e3684710be0c27903922553/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fsecurity%2Fsecurityexpression%2FUserSecurityExpression.java)|100%|:green_apple:|
|[UserService.java](https://github.com/itmo-events-app/event-app/blob/ba72ae724aa667cb1e3684710be0c27903922553/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fservice%2FUserService.java)|89.29%|:green_apple:|

File: src/main/java/org/itmo/eventapp/main/model/mapper/UserMapper.java (hunk 1 above)
Comment (Darklol) [reply src/main/java/org/itmo/eventapp/main/model/mapper/UserMapper.java:29]: Если аноним входи, то не факт, но у нас такого вроде нет

**Review by rimnvd — APPROVED** [review]

**Comment by IP-13** [comment]
|Overall Project|70.07%|:green_apple:|
|:-|:-|:-:|
|Files changed|100%|:green_apple:|
<br>

|File|Coverage||
|:-|:-|:-:|
|[ProfileController.java](https://github.com/itmo-events-app/event-app/blob/8b71ecb46ea72b8a6cfad91206c354e76ddd21f4/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fcontroller%2FProfileController.java)|100%|:green_apple:|
|[UserService.java](https://github.com/itmo-events-app/event-app/blob/8b71ecb46ea72b8a6cfad91206c354e76ddd21f4/src%2Fmain%2Fjava%2Forg%2Fitmo%2Feventapp%2Fmain%2Fservice%2FUserService.java)|89.29%|:green_apple:|

**Review by Darklol — APPROVED** [review]

**Pull request merged**